Add filename encode args

apolkingg8 opened the pull request:
fix issue #14

mceachen commented:
Thanks!

Would you be able to add an example image file and a test that reads that new file?

apolkingg8 commented:
Ok, I will add test for `read` / `write` non-english file.

apolkingg8 commented:
Note:
* Added `-charset filename=utf8` in both `read` and `write` default args for non-english filename/path.
* Added `-codedcharacterset=utf8` in `write` for non-english tag content
* Added their tests
@mceachen please check it :)
